osm2streets/src/intersection.rs

```diff
@@ -18,6 +18,8 @@ pub struct Intersection {
     /// StreetNetwork; roads and intersections get merged and deleted.
     pub point: Pt2D,
     /// This will be a placeholder until `Transformation::GenerateIntersectionGeometry` runs.
+    ///
+    /// TODO Consistently make this clockwise.
```

Comment:
Things like the sidewalk corner are... maybe?... sensitive to the polygon orientation. Anyway, once the dust settles, establishing this invariant would probably be helpful to downstream logic.